starter-code/src/viking.js

```diff
@@ -1,11 +1,111 @@
 // Soldier
-function Soldier () {}
+function Soldier (health, strength) {
+    this.health = health;
+    this.strength = strength;
+
+    Soldier.prototype.attack = function(){
+        return this.strength;
+    }
+
+    Soldier.prototype.receiveDamage = function(damage){
+      this.health -= damage;
+    }
+}
+
+
+var soldier1 = new Soldier(5,8);
+console.log(soldier1.attack());
+console.log(soldier1.receiveDamage(3));
 
 // Viking
-function Viking () {}
+function Viking (name,health,strength) {
+  Soldier.call(this,health,strength);
+  this.name= name;
+
+  Soldier.prototype.attack = function(){
+    return this.strength;
+  }
+
+  Viking.prototype.receiveDamage = function(damage){
+    this.health -= damage;
+    if(this.health > 0){
+      return this.name + ' has received ' +  damage + ' points of damage';
+    }else{
+      return this.name + ' has died in act of combat';
+    }
+  }
+
+  Viking.prototype.battleCry = function(){
+    return 'Odin Owns You All!!'
```

Comment:
Aquí te falla un test porque pide que el método devuelva 'Odin Owns You All!' en lugar de 'Odin Owns You All!!' :)